docs/grammar.txt

```diff
@@ -32,9 +32,14 @@ EventDefinition = 'event' Identifier IndexedParameterList 'anonymous'? ';'
 EnumValue = Identifier
 EnumDefinition = 'enum' Identifier '{' EnumValue? (',' EnumValue)* '}'
 
-IndexedParameterList = '(' ( TypeName 'indexed'? Identifier? (',' TypeName 'indexed'? Identifier?)* )? ')'
-ParameterList =        '(' ( TypeName            Identifier? (',' TypeName            Identifier?)* )? ')'
-TypeNameList =         '(' ( TypeName (',' TypeName )* )? ')'
+IndexedParameterList = '(' ( IndexedParameter (',' IndexedParameter )* )? ')'
+IndexedParameter = TypeName (StorageLocation | 'indexed')* Identifier?
```

Comment:
Hm I do not think it can be marked `storage indexed`. The `indexed` keyword is only for event arguments.

Reply:
You mean that `StorageLocation` is not allowed in event arguments, right? I was adding the `StorageLocation` to all parameter types and forgot about that. Will fix soon.

Reply:
I do not think it is allowed in events, if it is that should be a bug.

Reply:
Seemingly the parser doesn't allow it:
```
events.sol:2:16: Error: Expected token Comma got 'Storage'
  event A(uint storage a);
               ^
```

Reply:
Fixed.